package.json

```diff
@@ -19,7 +19,7 @@
         "@pxblue/colors": "^3.0.0",
         "@pxblue/icons-svg": "^1.1.1",
         "@pxblue/react-native-components": "5.0.0-beta.1",
```

Comment:
doesn't this need a component library update in order to work correctly as well?

Reply:
I got you covered